### ISSUE TITLE
Language request: Pony

### ISSUE THREAD

**Issue by ruuda** [AlDanial/cloc#697]
- language name: [Pony](https://www.ponylang.io/)
 - file extension(s): `.pony`
 - method(s) of commenting text: [C++ style](https://tutorial.ponylang.io/appendices/whitespace.html#comments): `//`-style line comments, and `/* */`-style block comments. Block comments can be nested.
 - location of sample code: https://github.com/ponylang/ponyc/tree/908402b60f083d66d3b59a1879efb6adee5ca3cc/examples

Aside, thank you for building Cloc! I’ve been happily using it for many years.

**Comment by AlDanial** [comment]
From looking at Pony examples such as https://github.com/ponylang/ponyc/blob/main/examples/ring/main.pony I saw it supports docstrings which can either be code or comments.  I gave Pony the same default behavior as other languages with docstrings like Python:  docstrings will be treated as comments.  If you want them counted as code, give cloc the  ``--docstring-as-code`` switch.

Let me know if you think that logic is wrong for Pony.

Also:  nested block comments will cause problems for cloc.  This is described in the Limitations section of the readme.

I'm pleased to hear that cloc as been useful to you!
